libvmi/driver/xen/xen_events.c

```diff
@@ -376,7 +376,7 @@ status_t process_unhandled_mem(vmi_instance_t vmi,
     mem_access_event_t event = { 0 };
     event.physical_address = page->key << 12;
     event.npages = 1;
-    xen_set_mem_access(vmi, &event, VMI_MEMACCESS_N);
+    xen_set_mem_access(vmi, &event, VMI_MEMACCESS_N, req->altp2m_idx);
```

Comment:
By "may be" what I meant there is that it "can be" used for altp2m. It's always 0 if VM_EVENT_ALTERNATE_P2M is not set.